nautobot/ui/src/views/Home.jsx

```diff
@@ -1,13 +1,11 @@
 import { Alert } from "@chakra-ui/react"
 
-
-export default function Home({ children }) {
+export default function Home() {
```

Comment:
Wasn't the `children` thing a leave over from embedding plugins (apps)?

Reply:
Not expressly. `children` is anything passed on the inside of the component. So:

```js
// App.jsx
<Home>
  <Button />
</Home>
```

`<Button />` become `children`.

However when we build the pluggable homepage these will likely need to be _either_ loaded directly in the `<Home>` component (not passed in) or at least passed in as named arguments, so

```js
// App.jsx
const LeftHomePanel = <Button />
const RightHomePanel = <Text />

<Home left=LeftHomePanel right=RightHomePanel />
```